pyfr/integrators/std/controllers.py

```diff
@@ -40,6 +41,12 @@ def _accept_step(self, dt, idxcurr, err=None):
         # Fire off any event handlers
         self.completed_step_handlers(self)
 
+        # Abort if plugins ask for it
```

Comment:
Also needs to be ported over to dual time stepping.

Reply:
Done